scripts/guest/m-clear-cache

```diff
@@ -60,7 +60,7 @@ if [[ -d "${MAGENTO_ROOT}/dev" ]]; then
     fi
 fi
 
-if [[ -f "${MAGENTO_ROOT}/app/etc/config.php" ]]; then
```

Comment:
@paliarush, please check this place. I suppose you used that file as a flag that Magento was installed before this script was run. If I'm right then a mark of that should be `env.php` because `config.php` could be persisted in git right after repository checkout, which is not what you expect.

Reply:
Exactly, at the time of the initial implementation we didn't have a recommendation to store config.php under git so it didn't matter too much.